cosmo_tester/framework/test_hosts.py

```diff
@@ -1105,7 +1115,9 @@ def _upload_blueprints_to_infrastructure_manager(self):
         self._logger.info(
             'Uploading test blueprints to infrastructure manager.'
         )
-        suffix = '-multi-net' if self.multi_net else ""
+        suffix = '-multi-net' if self.multi_net \
```

Comment:
How about just appending each of the possible suffixes so that making an ipv6 multi-net test later (if we decided to) would just be a matter of adding the blueprints and removing the check you put in above.

Reply:
(I think it'll also make this logic look a bit cleaner)